deepquest/qe_models/birnn_word.py

```diff
@@ -139,4 +140,9 @@ def build(self):
                 inputs=[src_words, trg_words],
                 outputs=[output_qe_layer]
                 )
+        
+        if params.get('N_GPUS', 1) > 1:
```

Comment:
How does this work? `N_GPUS` isn't set at the moment, is it?
I guess it should be in the `set_gpu_id` function.

Reply:
Yes, in this branch it's set in the config file, but you're right, we should set it automatically.